monitoring/alerts.yaml

```diff
@@ -8,6 +8,24 @@ x-inputs:
     value: artesca-data-connector-s3api-metrics
   - name: replicas
     type: constant
+  - name: systemErrorsWarningThreshold
+    type: constant
+    value: 0.03
+  - name: systemErrorsCriticalThreshold
+    type: constant
+    value: 0.05
+  - name: listingLatencyWarningThreshold
+    type: constant
+    value: 0.300
+  - name: listingLatencyCriticalThreshold
+    type: constant
+    value: 0.500
+  - name: deleteLatencyWarningThreshold
+    type: constant
+    value: 0.500
+  - name: deleteLatencyCriticalThreshold
+    type: constant
+    value: 1.000
```

Comment:
Why not `1`?

Reply:
not much: just "consistency" with the other values...